### DISCUSSION TITLE
Why no more .NET Framework target?

### DISCUSSION THREAD

**Discussion by catester** [dlemstra/Magick.NET#1230] [General]
I noticed we recently dropped net20 target. In my experience, referencing .NET Standard DLLs from a .NET Framework, does not work good. Would it be hard to support at least net461 as a base?

This is an example issue from another project:
https://github.com/adamhathcock/sharpcompress/pull/621

**Comment by dlemstra** [comment]
`net461` is end of life and it looks like the issues that are mentioned (a lot of extra dll's) can be resolved by upgrading to `net472`. Not sure if I want to restore a separate target framework for that. And to be honest you are the first one to ask for an explicit .NET Framework target.

**Reply by CommanderAlchemy** [reply]
@dlemstra 

I would like to ping here too, updated to 12.1.0 and I am getting build issues on my build server for a project with framework 4.8
Am I misunderstanding that .netstandard 2.0 should work on .NET Framework 4.8 for this package ?

```
Package Magick.NET-Q16-x64 12.1.0 is not compatible with net48 (.NETFramework,Version=v4.8). Package Magick.NET-Q16-x64 12.1.0 supports:
      - netstandard2.0 (.NETStandard,Version=v2.0)
      - netstandard2.1 (.NETStandard,Version=v2.1)
    One or more packages are incompatible with .NETFramework,Version=v4.8
```

**Reply by dlemstra** [reply]
You are probably using an old version of NuGet or VisualStudio that cannot restore the project properly.

**Reply by CommanderAlchemy** [reply]
Hm, 

> You are probably using an old version of NuGet or VisualStudio that cannot restore the project properly.

Worked successfully downgrading down to Magick.NET 11.3.0 for now, anything later and it fails with error above. 
But cannot trigger this in Rider so if maybe is something in msbuild/Nuget since Rider includes its own, thought first I thought it was some cache issue. Will see if I can trigger this locally using identical nuget/msbuild from buildserver.

Build server is using:
- Pool: Azure Pipelines
- Image: windows-latest
- NuGet Version: 4.1.0.2450
- MSBuild 17.3.1.41501 (Microsoft Visual Studio 2022 Enterprise)

**Reply by dlemstra** [reply]
Your NuGet version should be at least 4.3 (https://github.com/Azure/azure-functions-durable-extension/issues/428)

**Reply by CommanderAlchemy** [reply]
> Your NuGet version should be at least 4.3 ([Azure/azure-functions-durable-extension#428](https://github.com/Azure/azure-functions-durable-extension/issues/428))

Hi this worked, had to manually install nuget latest version on build server since apparently latest image and vs2022 is not enough.

But it magically worked with new version. Thank you!
